app/controllers/articles_controller.rb

```diff
@@ -1,9 +1,9 @@
 class ArticlesController < ApplicationController
 
-  http_basic_authenticate_with name: "dhh", password: "secret", except: [:index, :show]
+  before_action :authenticate_user!, except: [:index, :show]
 
 	def index
```

Comment:
this line move left for 4 space
## Note , can you config your editor 
ex : Sublimtext  - >  Preference -> Setting User 
```
{
	"auto_complete": true,
	"auto_complete_commit_on_tab": true,
	"copy_with_empty_selection": true,
	"ensure_newline_at_eof_on_save": true,
	"font_size": 15,
	"ignored_packages":
	[
		"Vintage"
	],
	"index_files": true,
	"rulers":
	[
		79
	],
	"save_on_focus_lost": true,
	"tab_size": 2,
	"translate_tabs_to_spaces": true,
	"trim_trailing_white_space_on_save": true
}

```